tutorials/export/exporting_basics.rst

```diff
@@ -12,36 +12,38 @@ just so they can open your project. Instead, you can *export* your project,
 converting it into a "package" that can be run by anyone.
 
 The way you export your game depends on what platform you are targeting. In
-this tutorial, you'll learn how to export the "Dodge the Creeps" game for a
+this tutorial, you'll learn how to export the *Dodge the Creeps* game for a
 variety of platforms. First, however, we need to make some changes to the
 way the game works.
 
-.. note:: If you haven't made "Dodge the Creeps" yourself yet, please read
+.. note:: If you haven't made *Dodge the Creeps* yourself yet, please read
           :ref:`doc_your_first_game` before continuing with this tutorial.
 
 Preparing the project
 ---------------------
 
```

Comment:
I feel like this whole section should go. All the way to to export preset. User wants to learn how to export a game. Knowing how to make "Dodge a Creep" is not required knowledge here. It doesn't make their ability to export any better or worse. Instead we should provide them with a export ready zip they can use to practice exporting and that way they can learn we save them reading while tone of docs unrelated to the export.